tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -16,6 +16,19 @@ function goHomeJoinNewGame() {
   cy.log('Joined new game successfully');
 }
 
+function validateMatch(match, length, p1, p2) {
+  expect(match.player1.id).to.eq(p1);
+  expect(match.player2.id).to.eq(p2);
+  expect(match.startTime).to.be.greaterThan(0);
+  expect(match.endTime).to.eq(null);
```

Comment:
It's surprising that endTime is always null. Are we only calling this on matches that are in progress? Or are matches not getting end times set when they should?

Reply:
The match isn't over yet, I dont think it is needed in every check probably just after the match was created and then it was copy pasted into the rest of the spots that checked.  I can refactor this to remove the end time and then just check for it after the match is created and then ended, that would reduce our code even slightly more